src/components/PdfHighlighter.tsx

```diff
@@ -397,9 +397,10 @@ export const PdfHighlighter = ({
       if (highlightBindings?.container?.isConnected) {
         renderHighlightLayer(highlightBindings, pageNumber);
       } else {
-        const { textLayer } =
```

Comment:
This annotationLayer just so happened to be there - I don't know enough about PDF.js (yet) to know if this is a sensible place to put things.

It did have attribute `hidden` set so I made sure to unset that elsewhere.